src/ui/button/button.tsx

```diff
@@ -0,0 +1,100 @@
+import styles from './button.module.css'
+
+import { FC, MouseEvent, ReactElement } from 'react'
+import cn, { Argument } from 'classnames'
+import _ from 'lodash'
+import { CSSProperties } from 'styled-components'
+
+export interface IButtonProps {
+  text?: string | JSX.Element
+  textSize?: 'xs' | 'sm'
+  size?: 'xs' | 'sm' | 'md'
+  disabled?: boolean
+  variant?:
+    | 'primary'
+    | 'secondary'
+    | 'secondary-dark'
+    | 'tertiary'
+    | 'primary-dark'
+    | 'diestruction'
```

Comment:
what it means, destruction? If it represents some reddish style, maybe better something like `danger`?

Reply:
<img width="622" alt="Screenshot 2022-05-19 at 10 37 30" src="https://user-images.githubusercontent.com/74919871/169226448-822bf10a-ba9e-40a0-9f88-cc8ddb4f730f.png">
That's from the Figma style system))

Reply:
Yep, it is name that used earlier and it used in figma